js/prefs.js

```diff
@@ -43,7 +43,7 @@ var prefs = new function Prefs() {
       indent_conditional: true,
     },
     'editor.lintDelay': 500,        // lint gutter marker update delay, ms
-    'editor.linter': 'csslint',     // 'csslint' or 'stylelint' or ''
+    'editor.linter': 'stylelint',     // 'csslint' or 'stylelint' or ''
     'editor.lintReportDelay': 4500, // lint report update delay, ms
```

Comment:
Oh, I didn't mean to make stylelint the overall default. It can be a bit slow in some situations, so keeping 'csslint' the default would be best. I meant to switch it to stylelint when the editor is in usercss mode &ndash; maybe only when `@preprocessor stylus` is active. Thoughts @tophf?

Reply:
So we have to switch the linter without changing the prefs?

Reply:
Since CSSLint is useless for this particular case, it obviously makes sense to enforce Stylelint in usercss editor (if linter is not disabled).